docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -0,0 +1,424 @@
+---
+title: Managing Compute Resources for Containers
+---
+
+{% capture overview %}
+
+When you specify a [Pod](/docs/user-guide/pods), you can optionally specify how
+much CPU and memory (RAM) each Container needs. When Containers have resource
+requests specified, the scheduler can make better decisions about which nodes to
+place Pods on. And when Containers have their limits specified, contention for
+resources on a node can be handled in a specified manner. For more details about
+the difference between requests and limits, see
+[Resource QoS](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/resource-qos.md).
+
+{% endcapture %}
+
+
+{% capture body %}
+
+## Resource types
+
+*CPU* and *memory* are each a *resource type*. A resource type has a base unit.
+CPU is specified in units of cores, and memory is specified in units of bytes.
+
+CPU and memory are collectively referred to as *compute resources*, or just
+*resources*. Compute
+resources are measurable quantities that can be requested, allocated, and
+consumed. They are distinct from
+[API resources](/docs/api/). API resources, such as Pods and
+[Services](/docs/user-guide/services) are objects that can be read and modified
+through the Kubernetes API server.
+
+## Resource requests and limits of Pod and Container
+
+Each Container of a Pod can specify one or more of the following:
+
+* `spec.containers[].resources.limits.cpu`
+* `spec.containers[].resources.limits.memory`
+* `spec.containers[].resources.requests.cpu`
+* `spec.containers[].resources.requests.memory`.
+
+Specifying resource requests and limits is optional. In some clusters, unset
+limits or requests might be replaced with default values when a Pod is created
+or updated. The default values depends on how the cluster is configured. If the
+request values are not specified, they are set to be equal to the limit values
+by default. Note that limits must always be greater than or equal to requests.
+
+Although requests and limits can only be specified on individual Containers, it
+is convenient to talk about Pod resource requests and limits. A
+*Pod resource request/limit* for a particular resource type is the sum of the
+resource requests/limits of that type for each Container in the Pod, with
+unset values treated as zero (or equal to default values in some cluster
```

Comment:
What type of cluster configurations would lead to unset values being set to default values instead of being treated as zero? These are quite different outcomes, right? Is there somewhere else that can be linked to where this is described in more detail?